security.php

```diff
@@ -18,6 +18,7 @@
 
 define( 'CACHE_GROUP_LOGIN_LIMIT', 'login_limit' );
 define( 'CACHE_GROUP_LOST_PASSWORD_LIMIT', 'lost_password_limit' );
+define( 'CACHE_KEY_LOCK_PREFIX', 'locked_' );
```

Comment:
```suggestion
define( 'CACHE_KEY_LOCK_PREFIX', 'vip_locked_' );
```

I wonder if adding a VIP prefix to this would help namespace / deliminate it from other plugins who may cache values  with a `locked_` prefix.

Reply:
I guess it would. But it would break the convention of the existing count keys that don't have `vip_` prefix. Please see description which I now filled for details.

Reply:
What's the downside of switching? A period of up to 30 minutes where the count may be off (people get a few extra tries to log in when the new key name gets deployed), and a need to delete the old key names on a request (code can be removed in the next deploy).

(The PHP constant should also ideally be namespace / prefixed to reduce the risk of clashing with another plugin.)

Reply:
> But it would break the convention of the existing count keys that don't have vip_ prefix

So we would have `127.0.0.1|batman` and `vip_locked_127.0.0.1|batman` cache keys. One with a prefix and the other without.

However, I now prefixed the cache groups. Is that covering your worry of the naming clash?

Reply:
Yes ✅